gui/src/main/java/com/devonfw/ide/gui/IdeGuiContext.java

```diff
@@ -0,0 +1,34 @@
+package com.devonfw.ide.gui;
+
+import java.nio.file.Path;
+
+import com.devonfw.tools.ide.context.AbstractIdeContext;
+import com.devonfw.tools.ide.context.IdeStartContextImpl;
+import com.devonfw.tools.ide.io.IdeProgressBar;
+
+/**
+ *
+ */
+public class IdeGuiContext extends AbstractIdeContext {
+
+
+  /**
+   * The constructor.
+   *
+   * @param startContext the {@link com.devonfw.tools.ide.log.IdeLogger IdeLogger}.
```

Comment:
did you somehow copy that from `AbstractIdeContext` where it was a leftover from an incomplete refactoring?
```suggestion
   * @param startContext the {@link IdeStartContextImpl}.
```
You can also remove the `Impl` suffix in the JavaDoc...